stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -1970,4 +1957,58 @@ end
 (*)(A::SparseVecOrMat{Float64,Ti},
     B::Hermitian{Float64,SparseMatrixCSC{Float64,Ti}}) where {Ti} = sparse(Sparse(A)*Sparse(B))
 
+# Sort all the indices in each column for the construction of a CSC sparse matrix
+# sortBuffers!(A, sortindices = :sortcols)        # Sort each column with sort()
+function sortBuffers!(m, n, colptr::Vector{Ti}, rowval::Vector{Ti}, nzval::Vector{Tv}) where {Ti <: Integer, Tv}
```

Comment:
Not, but this looks a bit non-standard to me, any special reason for the camel case?

Reply:
You're right, this comes from `sortSparseMatrixCSC` that was in sparsematrix.jl (which I didn't even remove for the moment), I'll rename it (it should not be exported though).